pkg/executor/sortexec/sort.go

```diff
@@ -357,20 +596,18 @@ func (e *SortExec) buildKeyColumns() {
 	}
 }
 
-func (e *SortExec) lessRow(rowI, rowJ chunk.Row) bool {
+func (e *SortExec) lessRow(rowI, rowJ chunk.Row) int {
```

Comment:
Why modify this function?

Reply:
> Why modify this function?

Because `slices.SortFunc` is faster and it needs a compare function returning int type.